### ISSUE TITLE
Error code Sub-process dpkg --set-selections returned an error code (2)

### ISSUE THREAD

**Issue by Tropicaleb** [checkra1n/BugTracker#194]
**What device + iOS version are you on?**
iPhone 6 ios 12.4.1

**What checkra1n version are you using?**
1.0
**What are the steps to reproduce the issue?**
1. Open cydia
2. Upgrade or install or delete any tweaks 
...

**What do you expect, and what is happening instead?**
I expect my tweaks to be downloaded and upgraded but I get the error mentioned in the title

**Any other info, error logs, screenshots, ...?**
Under the "Sub-process dpkg --set-selections returned an error code (2)" error, I have "Couldn't record the approved state changes as dpkg selection states.

Thanks for your help.

**Comment by TysonSchultz51345** [comment]
i cant even open cydia

**Comment by Tropicaleb** [comment]
UPDATE

After restoring the system through checkra1n loader, it seemed that the upgrading worked until the end where I get the error in red "DPGK LOCKED".

**Comment by nullpixel** [comment]
were you already jailbroken with unc0ver?

**Comment by Tropicaleb** [comment]
@nullpixel No. The last jailbreak I had was on ios 10 with Yalu. But I restored and updated my iPhone two months ago so my phone was not jailbroken during the process.

**Comment by bansdude** [comment]
@nullpixel I have the same error, and I was jailbroken with unc0ver

**Comment by FranciscoKP2246** [comment]
I have the same issue, I hit the button where it says “restore system” within checkra1n because of a bad tweak then I tried to jailbreak again and now when I use Cydia it says DPGK Locked

**Comment by csc-EricWu** [comment]
me too
